laterpay/asset_sources/scss/components/_purchase_button.scss

```diff
@@ -90,6 +90,13 @@
     }
 }
 
+// wrapper used to set the purchase button apart from the post content when positioned automatically
+.lp_purchase-button-wrapper {
```

Comment:
Selector `lp_purchase-button-wrapper` should be written in lowercase with hyphens